### PR TITLE
fix: don't crash on mongodb host querystring params

### LINKED ISSUE #16: mongodb+srv urls with querystring parameters cause the forum to fail
mongodb+srv urls are introduced in this PR: https://github.com/overhangio/tutor-forum/pull/10
But querystring parameters will not work in the forum because of the way the `MONGOHQ_URL`variable is generated:

    MONGOHQ_URL="$MONGODB_HOST/$MONGODB_DATABASE"

We need to modify the definition of `MONGOHQ_URL` such that querystring parameters are supported. If possible, we should take this opportunity to get rid of the forum entrypoint entirely.

### PR DESCRIPTION
**Pull request by regisb**
When the MONGODB_HOST url contained querystring parameters, the forum was failing because the outdated mongodb gem does not support them. So we just trim them from the setting.

Close #16.

### CODE REVIEW

**Comment by regisb** [comment]
Cc @keithgg.

File: tutorforum/plugin.py [hunk 1]
```diff
@@ -75,11 +87,31 @@ def auth_mech_as_ruby(auth_mech):
...
+    # 0 = scheme
+    # 1 = netloc
+    # 2 = path
+    # 3 = params
+    # 4 = query
+    # 5 = fragment
+    parsed = [*urllib.parse.urlparse(host)]
+    parsed[4] = ""
+    return urllib.parse.urlunparse(parsed)
+
```
[the remark this answers is not in the document]
Comment (ghassanmas) [reply tutorforum/plugin.py:108]: This lead to an error as expected, I think one suggestion to resolve it; 
```
    parsed = [*urllib.parse.urlparse(host)]
    parsed[2] = "" 
     parsed[4] = ""
    return urllib.parse.urlunparse(parsed)
```

We don't expect the path to be set, given in mongo it refer to the database name which is handled/set by docker-entrypoint.sh.

[the remark this answers is not in the document]
Comment (regisb) [reply tutorforum/plugin.py:108]: I think I'll just strip the trailing slash from the path.

[the remark this answers is not in the document]
Comment (regisb) [reply tutorforum/plugin.py:111]: Thanks for the precise review! I believe that the proposed change resolves this issue.

**Review by ghassanmas — APPROVED** [review]
Tested (As local service and remote mongo cluster) and all good  👍

**Pull request merged**